src/inspircd.cpp

```diff
@@ -578,21 +583,29 @@ InspIRCd::InspIRCd(int argc, char** argv) :
 
 void InspIRCd::UpdateTime()
 {
-#ifdef _WIN32
+#if defined __MACH__
+	clock_serv_t cs;
+	host_get_clock_service(mach_host_self(), CALENDAR_CLOCK, &cs);
+	mach_timespec_t mts;
+	clock_get_time(cs, &mts);
+	mach_port_deallocate(mach_task_self(), cs);
```

Comment:
Is there a need to do the `host_get_clock_service()` and `mach_port_deallocate()` calls every UpdateTime()? Wouldn't it be more wiser to do it once on startup and shutdown?